server/src/modules/rules/constants/rules.constants.ts

```diff
@@ -83,6 +83,38 @@ export class RuleType {
     [RulePossibility.EQUALS, RulePossibility.NOT_EQUALS],
     'boolean',
   );
+  static readonly NUMBER_LIST = new RuleType(
+    '4',
+    [
+      RulePossibility.EQUALS,
+      RulePossibility.NOT_EQUALS,
+      RulePossibility.CONTAINS,
+      RulePossibility.NOT_CONTAINS,
+      RulePossibility.CONTAINS_PARTIAL,
+      RulePossibility.NOT_CONTAINS_PARTIAL,
+      RulePossibility.COUNT_EQUALS,
+      RulePossibility.COUNT_NOT_EQUALS,
+      RulePossibility.COUNT_BIGGER,
+      RulePossibility.COUNT_SMALLER,
+    ],
+    'number list',
+  );
+  static readonly TEXT_LIST = new RuleType(
```

Comment:
I just found that it's currently not possible to save a custom value for these types. It looks like there's a couple of functions that switch on the types to handle custom values:
See `server\src\modules\rules\constants\constants.service.ts`
`getCustomValueIdentifier`
`getCustomValueFromIdentifier`

Reply:
aha good catch, thanks, just realised I didn't fully test end-to-end yet - only to the point of selecting it in the UI. I'll also actually run it on some rules and make sure it works...

Reply:
Seems working fully:

![image](https://github.com/user-attachments/assets/d1094f30-99a3-4596-ae57-62a4413e775c)
![image](https://github.com/user-attachments/assets/d7eebe76-e29b-4d32-91dc-6558ac40e040)
![image](https://github.com/user-attachments/assets/bc7a16cd-7701-45f7-afac-e60265aa136e)

Reply:
The non-count lists do not though, e.g.:
![image](https://github.com/user-attachments/assets/8c69def5-618a-4563-9f06-5d9f7bf67070)